jupyter-lib/api/src/main/kotlin/org/jetbrains/kotlinx/jupyter/api/KotlinKernelHost.kt

```diff
@@ -34,4 +34,9 @@ interface KotlinKernelHost {
      * Adds a new library via its definition. Fully interchangeable with `%use` approach
      */
     fun addLibrary(library: LibraryDefinition)
+
+    /**
+     * Declares the given variables. They may be used in the notebook cells
+     */
+    fun declareVariables(variables: Iterable<VariableDeclaration>)
```

Comment:
I think it is better to call them properties since they are not actually variable.